File: C++/500. Keyboard Row.cpp
// Time: O(n*m)
class Solution {
public:
    vector<string> findWords(vector<string>& words) {
        vector<string> cnt {"qwertyuiop", "asdfghjkl", "zxcvbnm"};
        vector<string> ans;
        int len = words.size();
        for(int i=0;i<len;i++)
        {
            
            int d[3] = {0,0,0};
            for(int j=0;j<words[i].length();j++)
            {
                for(int k=0;k<3;k++)
                {
                    if(cnt[k].find(tolower(words[i][j]))!=-1)
                        d[k] = 1;
                }
            }
            if(d[0]+d[1]+d[2]==1)
                ans.push_back(words[i]);
        }
        return ans;
    }
};

File: Python/500. Keyboard Row.py
# Time: O(n*m) 
class Solution:
    def findWords(self, words):
        """
        :type words: List[str]
        :rtype: List[str]
        """
        key = ['qwertyuiop', 'asdfghjkl', 'zxcvbnm']
        ans = list()
        for word in words:
            k = [1, 1, 1]
            for s in word:
                for i in range(len(key)):
                    if s.lower() in key[i]:
                        k[i] = 0
            if k[0] + k[1] + k[2] == 2:
                ans.append(word)
        return ans